### PR TITLE
Add user to answers

### PR DESCRIPTION
**Pull request by JustRoxy**

### CODE REVIEW

File: backend/CircleForms/Models/Posts/Answer.cs [hunk 1]
```diff
@@ -1,22 +1,35 @@
...
+public class Answer : Entity
 {
     [JsonProperty("answers")]
     public List<Submission> Submissions { get; set; }
 
-    [BsonId]
-    [JsonProperty("user_id")]
-    public string ID { get; set; }
+    [JsonProperty("user")]
+    public One<User> User { get; set; }
```
Comment (stanriders) [inline backend/CircleForms/Models/Posts/Answer.cs:16]: Why can't we do the same for forms?

File: backend/CircleForms/Contracts/ContractModels/Response/AnswerContract.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using CircleForms.Models.Posts.Questions.Submissions;
+using Newtonsoft.Json;
+
+namespace CircleForms.Contracts.ContractModels.Response;
+
+public class AnswerContract
+{
+    [JsonProperty("answers")]
```
Comment (stanriders) [inline backend/CircleForms/Contracts/ContractModels/Response/AnswerContract.cs:9]: Might be a good idea to rename this to `submissions`

**Review by stanriders — CHANGES_REQUESTED** [review]
I assume this doesn't blow up the existing database

File: backend/CircleForms/Models/Posts/Answer.cs (hunk 1 above)
Comment (JustRoxy) [reply backend/CircleForms/Models/Posts/Answer.cs:16]: because I remembered about it 1 day ago

**Comment by JustRoxy** [comment]
> I assume this doesn't blow up the existing database

it shouldn't

File: backend/CircleForms/Models/Posts/Answer.cs (hunk 1 above)
Comment (stanriders) [reply backend/CircleForms/Models/Posts/Answer.cs:16]: I think it might be worth doing in a separate PR

**Review by stanriders — APPROVED** [review]

**Pull request merged**